frontend/www/js/omegaup/arena/admin_arena.ts

```diff
@@ -74,45 +74,49 @@ export default class ArenaAdmin {
   }
 
   setUpPagers(): void {
-    $('.clarifpager .clarifpagerprev').on('click', () => {
-      if (this.arena.clarificationsOffset > 0) {
-        this.arena.clarificationsOffset -= this.arena.clarificationsRowcount;
+    document
+      .querySelector('.clarifpager .clarifpagerprev')
+      ?.addEventListener('click', () => {
+        if (this.arena.clarificationsOffset > 0) {
+          this.arena.clarificationsOffset -= this.arena.clarificationsRowcount;
+          if (this.arena.clarificationsOffset < 0) {
+            this.arena.clarificationsOffset = 0;
+          }
+
+          this.refreshClarifications();
+        }
+      });
+
+    document
+      .querySelector('.clarifpager .clarifpagernext')
+      ?.addEventListener('click', () => {
+        this.arena.clarificationsOffset += this.arena.clarificationsRowcount;
         if (this.arena.clarificationsOffset < 0) {
           this.arena.clarificationsOffset = 0;
         }
 
         this.refreshClarifications();
-      }
-    });
+      });
 
-    $('.clarifpager .clarifpagernext').on('click', () => {
-      this.arena.clarificationsOffset += this.arena.clarificationsRowcount;
-      if (this.arena.clarificationsOffset < 0) {
-        this.arena.clarificationsOffset = 0;
+    this.arena.elements.clarification?.addEventListener('submit', () => {
+      const clarification = this.arena.elements.clarification;
```

Comment:
súper nit, pero si se puede

```suggestion
      const clarificationElement = this.arena.elements.clarification;
```

esto ayudaría a distinguir entre una clarificación y el _elemento_ de clarificaciones.